apps/spruce/src/components/Spawn/utils/hostUptime.test.ts

```diff
@@ -549,3 +550,89 @@ describe("getNextHostStart", () => {
     });
   });
 });
+
+describe("getEnabledHoursCount", () => {
```

Comment:
Are there any possible invalid inputs this function could get passed for example `stopTime` is before `startTime` that we should test against.

Reply:
`stopTime` before `startTime` is exactly what this PR implements to allow overnight schedules 😅